app/views/play/common/ChangeLanguageTab.js

```diff
@@ -54,9 +55,10 @@ module.exports = (ChangeLanguageTab = (function () {
       context = super.getRenderData(context)
       context.codeLanguages = this.codeLanguageList
       context.codeLanguage = this.codeLanguage = this.options?.session?.get('codeLanguage') || me.get('aceConfig')?.language || 'python'
-      context.codeFormats = this.codeFormatList
+      context.codeFormats = this.codeFormatList.filter(({ id }) => (this.classroomAceConfig?.codeFormats || ['text-code', 'blocks-and-code', 'blocks-text', 'blocks-icons']).includes(id))
+      console.log('codeFormatList', context.codeFormats)
```

Comment:
you should remove logs in general while creating PR

Reply:
oh ya. normally i do that. not sure why I forget this time 😂 maybe I should try to make a pre-commit hook 😂